messenger/working-with-doctrine.rst

```diff
@@ -0,0 +1,45 @@
+.. index::
+    single: Messenger; Working with Doctrine
+
+Working with Doctrine
+=====================
+
+If your message handlers writes to a database it is a good idea to wrap all those
+writes in a single Doctrine transaction. This make sure that if one of your database
+query fails, then all queries are rolled back and give you a change to handle the
+exception knowing that your database was not changed by your message handler.
+
+To make sure your message bus wraps the handler in one transaction you must first
```

Comment:
The following section can now be removed since the PR on DoctrineBundle is merged